browser/src/control/Control.Menubar.js

```diff
@@ -1723,7 +1723,7 @@ L.Control.Menubar = L.Control.extend({
 			this._map.openSaveAs();
 		} else if (id === 'exportpdf') {
 			this._map.dispatch('exportpdf');
-		} else if (id === 'exportepub') {
+		} else if (id === 'exportepub' || id === 'downloadas-epub') {
```

Comment:
I would recommend to rather change 'downloadas-epub' id to 'exportepub' in this file above so we will be unified with desktop variant. Then it will be handled only in one place here and not below in the "downloadas-"

Reply:
and the same for pdf (it also should show the dialog)

Reply:
Yes make sense, I'll try